olp-cpp-sdk-dataservice-read/tests/StreamApiTest.cpp

```diff
@@ -39,6 +39,22 @@ std::string ApiErrorToString(const ApiError& error) {
   return result_stream.str();
 }
 
+model::StreamOffsets GetStreamOffsets() {
+  static model::StreamOffsets offsets;
```

Comment:
Why does it need to be static?

Reply:
To be initialized and filled only once. It will be used for SeekToOffset tests also

Reply:
This is just complicating the test. You make static something that needs to have a certain lifecycle or has some limitations that force you to use the singleton pattern. In this scenario, I don't see any need to make it static and this only adds additional confusion.
Is there any limitation that forces you to make it static or do you aim for obscurity?

Reply:
Static variable is removed.